docs/content/querying/filters.md

```diff
@@ -81,6 +81,35 @@ The following matches any dimension values for the dimension `name` between `'ba
 }
 ```
 
+The following matches values of the given two dimensions `dim1` and `dim2` are the same
+```json
+{
+  "type" : "javascript",
+  "dimensions" : ["dim1","dim2"]
+  "function" : "function(x, y) { return x === y }"
+}
+
+### JavaScript2 filter
+
+The JavaScript2 filter is basically the same with JavaScript filter but it differs in that it supports multiple dimensions and multi-valued column is handed over as single object array. And also, this filter does not use bitmap index and depends on full scan of data.
+
+```json
+{
+  "type" : "javascript2",
```

Comment:
I think it is better to have a flag on javascript filter that utimately leads to two code paths, one that uses indexes, and one that doesnt rather htan having two separate implementations. Two impls will confuse users.